### PR TITLE
feat_FE_googleMapMarker

### PR DESCRIPTION
**Pull request by davidaceves**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

A marker is placed on the map corresponding with nearby search results. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.
- [x] Refactor
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status
- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: client/src/components/Map/Map.jsx [hunk 1]
```diff
@@ -94,7 +76,7 @@ class Map extends Component {
       location: place.geometry.location,
       placeId: place.place_id,
       radius: '500',
-      query: "cafe"
+      query: "Cafe"
```
Comment (MrT3313) [inline client/src/components/Map/Map.jsx]: What other categories are there? is there a way to also query for specific names and add that to the list that is returned? like Starbucks / McDonalds / Dunkin Donuts etc etc?

Comment (davidaceves) [reply client/src/components/Map/Map.jsx]: There is a list of locations here: https://tinyurl.com/yxtm5qns. It is also possible to make the query dynamic based on whatever the user is looking for, you would just need to connect it to the input box.

Comment (Memitaru) [reply client/src/components/Map/Map.jsx]: Can the base query do multiple categories? I feel like libraries would be good to include.

File: client/src/components/Map/MAPMANUALTESTING.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+GIVEN I am a user
+    WHEN I navigate to the home
+    AND I see the map component
+    AND I see the geo-location alert
+    AND I click cancel
+    THEN I see the map has been redirected to Sydney Australia
+
+GIVEN I am a user
```
Comment (Memitaru) [inline client/src/components/Map/MAPMANUALTESTING.md]: I really like the details on manual testing. Gives me a good idea of what behaviors we should be able to see.

**Pull request merged**